src/pages/HistoryPage.spec.js

```diff
@@ -0,0 +1,30 @@
+import { render, screen } from '@testing-library/react'
+import HistoryPage from './HistoryPage'
+
+describe('HistoryPage', () => {
+  it('has a list of HistryEntrys', () => {
+    render(
+      <HistoryPage
+        history={[
+          {
+            id: '01',
+            playerName: 'John Doe',
+            date: '17. Juni 2021',
+            playlistName: 'Italo Disco 80s',
+            score: 1123,
+          },
+          {
+            id: '02',
+            playerName: 'Jane Doe',
+            date: '17. Juni 2021',
+            playlistName: 'Italo Disco 80s',
+            score: 1312,
+          },
+        ]}
+      />
+    )
+
+    const listItems = screen.getAllByRole('listitem')
+    expect(listItems.length).toBeGreaterThan(1)
```

Comment:
looks a little unspecific at first but maybe it is enough... so maybe you want to consider to make the test a little more specific or you leave it like it is 😉

Reply:
Hm but the page didn't contain much more then a list of elements, and the elements are tested on his own. How would you specify this test?